gammapy/cube/tests/test_core.py

```diff
@@ -133,8 +133,8 @@ def test_integral_flux_image(self):
         assert_allclose(new_image.data, image.data)
 
         # Test Header Keys
-        expected = [('CDELT1', 0.5), ('CDELT2', 0.5), ('NAXIS1', 61),
-                    ('NAXIS2', 21), ('CRVAL1', 0), ('CRVAL2', 0)]
```

Comment:
Yes, that's right. The `NAXIS` Keywords are only filled (automatically) when the data is written to file.  For the `reproject_to` method it might be better to use `meta.update(header_out)` i.e. update the keywords that have changed and just copy the rest, instead  of only filling it with the information of `header_in` only.